app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,153 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+/* General tag styles */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #f5f6f8;
+  font-family: 'Quicksand', sans-serif;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: gray;
+}
+
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+/* rails styles modifications */
+
+.field {
+  width: 100%;
+}
+
+.actions {
+  width: 100%;
+}
+
+/* general styles */
+
+.w-100 {
+  width: 100%;
+}
+
+.separator-line {
+  border: 1px solid lightgrey;
+  width: 100%;
+  margin: 0;
+}
+
+.mobile-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: white;
+  padding: 15px;
+  margin-top: 50px;
+}
+
+.mobile-field {
+  width: 100%;
+  height: 40px;
+  border: 0;
+}
+
+.mobile-field::placeholder {
+  color: gray;
+}
+
+.mobile-button-blue {
+  width: 100%;
+  height: 40px;
+  color: white;
+  border: 0;
+  background-color: #3778c2;
+}
+
+.mobile-button-white {
+  width: 100%;
+  height: 40px;
+  color: gray;
+  border: 0;
+  background-color: white;
+}
+
+/* views styles */
+
+.splash-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 90vh;
+  align-items: center;
+  justify-content: center;
+}
+
```

Comment:
* As stated in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Typography%3A%20font%20face,space%20between%20elements.) Kindly follow the Typography: font face, size and weight.
Layout: composition and space between elements.  your text text color and font family  should match that of the design and your CTA buttons too 
![image](https://user-images.githubusercontent.com/52497246/235037235-6e2e8725-cdc7-4bbc-8e35-7c7d0142bfad.png)
![image](https://user-images.githubusercontent.com/52497246/235037441-aa82195a-1233-4f12-a1cc-a9bbcb6fc208.png)
![image](https://user-images.githubusercontent.com/52497246/235037494-145e2ba1-c62d-4e45-947f-298b1d3aa2dd.png)